go/vt/vttablet/tabletserver/vstreamer/rowstreamer.go

```diff
@@ -119,7 +119,7 @@ func (rs *rowStreamer) Stream() error {
 		return err
 	}
 	defer conn.Close()
-	if _, err := conn.ExecuteFetch("set names binary", 1, false); err != nil {
+	if _, err := conn.ExecuteFetch("set names 'binary'", 1, false); err != nil {
```

Comment:
why is this change needed?

Reply:
then, its clearly a parsing bug. :(

Reply:
That may be, but the "correct" syntax is to use quotes anyway so I did not find it relevant to my PR and instead used the quotes.

https://dev.mysql.com/doc/refman/8.0/en/set-names.html